.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
I raised this with the last dependabot config, but why is this `pip` and not `poetry`?

Reply:
Basically because poetry uses pip